core/src/test/resources/org/apache/calcite/test/RelOptRulesTest.xml

```diff
@@ -8177,7 +8195,8 @@ LogicalProject(DEPTNO=[$0])
   LogicalJoin(condition=[true], joinType=[inner])
     LogicalTableScan(table=[[scott, DEPT]])
     LogicalAggregate(group=[{}], DUMMY_COUNT=[COUNT()])
-      LogicalTableScan(table=[[scott, EMP]])
+      LogicalProject(DUMMY=[0])
```

Comment:
Do you know why we have this extra `Project` now?

Reply:
Yes. Different `RelBuilder` configuration instances are used at different phase of the test. 
[RelBuilder.Config.DEFAULT](https://github.com/apache/calcite/blob/979981481f8effde5e2fe5bb605d5c3424554e4b/core/src/main/java/org/apache/calcite/tools/RelBuilder.java#L4925-L4927) vs [RelOptFixture.DEFAULT](https://github.com/apache/calcite/blob/979981481f8effde5e2fe5bb605d5c3424554e4b/testkit/src/main/java/org/apache/calcite/test/RelOptFixture.java#L88)
`pruneInputOfAggregate` has different values in the two instances. It would be good to use only one instance in the whole test but I haven't been able to achieve this so far.

Reply:
Reverted this change.